packages/tools/src/stateManagement/annotation/annotationState.ts

```diff
@@ -40,58 +39,59 @@ function getViewportSpecificAnnotationManager(
   return defaultFrameOfReferenceSpecificAnnotationManager;
 }
 
+function getAnnotationManager(element?: HTMLDivElement) {
+  if (element) {
+    return getViewportSpecificAnnotationManager(element);
+  }
+
+  return getDefaultAnnotationManager();
+}
+
 /**
- * Returns the annotations for the `FrameOfReference` of the `Viewport`
- * being viewed by the cornerstone3D enabled `element`.
+ * Returns the annotations for a given tool in the specified frame of reference.
+ * If element is provided, it will return the annotations for the viewport specific
+ * annotation manager.
  *
- * @param element - The HTML element.
  * @param toolName - The name of the tool.
+ * @param FrameOfReferenceUID - The Frame of Reference UID.
+ * @param element - The HTML element.
  * @returns The annotations corresponding to the Frame of Reference and the toolName.
  */
 function getAnnotations(
-  element: HTMLDivElement,
-  toolName: string
+  toolName: string,
+  FrameOfReferenceUID: string,
+  element?: HTMLDivElement
 ): Annotations {
-  const enabledElement = getEnabledElement(element);
-  const annotationManager =
-    getViewportSpecificAnnotationManager(enabledElement);
-  const { FrameOfReferenceUID } = enabledElement;
-
+  const annotationManager = getAnnotationManager(element);
   return annotationManager.get(FrameOfReferenceUID, toolName);
 }
 
 /**
- * Add the annotation to the annotations for the `FrameOfReference` of the `Viewport`
- * being viewed by the cornerstone3D enabled `element`.
+ * Add the annotation to the annotation manager. If an element is provided,
+ * the annotation will be added to the viewport specific annotation manager.
  *
- * @param element - HTMLDivElement
  * @param annotation - The annotation that is being added to the annotations manager.
+ * @param element - HTMLDivElement
  */
 function addAnnotation(
-  element: HTMLDivElement,
-  annotation: Annotation
+  annotation: Annotation,
+  element?: HTMLDivElement
```

Comment:
Suggest a parameter "managerSelector" that has various types associated with it that we can apply later to figure out the right manager.